### PR TITLE
Memstick screen updates (scoped storage)

### PR DESCRIPTION
**Pull request by hrydgard**
This doesn't redesign it much visually (will be in later PR) but does some tweaks around it:

 * Adds a Help button linking to the website
 * Shows the LogoScreen before the first-boot memstick screen
 * Avoids moving large ISO files when you move the memstick directory, if they can't be insta-moved. Not sure if it should be an option...
 * Moves the memstick options higher on the System tab, to make them easier to find.
 * New icons for folder and warning

LATEST:

Screenshot_20210919-192116

OLD:

image

### CODE REVIEW

**Comment by unknownbrackets** [comment]
> Avoids moving large ISO files when you move the memstick directory, if they can't be insta-moved. Not sure if it should be an option...

Hm.  Speaking of, what happens with Recent, I wonder?  Do we need to change those paths for things we do move?

I can imagine it being confusing for ISOs not to move, but I can't really think of a strong scenario where it'd be desirable to move them, indeed...

-[Unknown]

File: UI/MemStickScreen.cpp [hunk 1]
```diff
@@ -176,11 +177,18 @@ void MemStickScreen::CreateViews() {
...
 	if (!initialSetup_) {
 		rightColumnItems->Add(new Choice(di->T("Back")))->OnClick.Handle<UIScreen>(this, &UIScreen::OnBack);
+		rightColumnItems->Add(new Choice(di->T("Help")))->OnClick.Handle<MemStickScreen>(this, &MemStickScreen::OnHelp);
 	}
 
 	INFO_LOG(SYSTEM, "MemStickScreen: initialSetup=%d", (int)initialSetup_);
 }
 
+UI::EventReturn MemStickScreen::OnHelp(UI::EventParams &params) {
+	LaunchBrowser("https://ppsspp.org/guide_storage.html");
```
Comment (unknownbrackets) [inline UI/MemStickScreen.cpp:187]: nit: It's better for SEO if you use a consistent hostname (and scheme.)  Elsewhere www is used, and search engines consider them completely separate URLs - for example if 1000 people link to "https://ppsspp.org/" and 1000 people link to "https://www.ppsspp.org/", both sites are considered individually less popular than one that is linked to by 2000 people.  It's more complex than that, but at a basic level.

-[Unknown]

File: UI/MemStickScreen.cpp [hunk 2]
```diff
@@ -510,12 +525,17 @@ UI::EventReturn ConfirmMemstickMoveScreen::OnConfirm(UI::EventParams &params) {
...
 
 			bool dryRun = false;  // Useful for debugging.
 
-			size_t moveFailures = 0;
+			size_t failedFiles = 0;
+			size_t skippedFiles = 0;
+
+			// We're not moving huge files like ISOs during this process, unless
+			// they can be directly moved, without rewriting the file.
+			const uint64_t BIG_FILE_THRESHOLD = 24 * 1024 * 1024;
```
Comment (unknownbrackets) [inline UI/MemStickScreen.cpp:538]: Hm, install and saved replay movie data in some games might exceed this and confuse users if not moved.

-[Unknown]

File: UI/MemStickScreen.cpp [hunk 3]
```diff
@@ -176,11 +177,18 @@ void MemStickScreen::CreateViews() {
 
 	if (!initialSetup_) {
 		rightColumnItems->Add(new Choice(di->T("Back")))->OnClick.Handle<UIScreen>(this, &UIScreen::OnBack);
+		rightColumnItems->Add(new Choice(di->T("Help")))->OnClick.Handle<MemStickScreen>(this, &MemStickScreen::OnHelp);
```
Comment (iota97) [inline UI/MemStickScreen.cpp:180]: We should always show the "Help" button, even on first run.

File: UI/MemStickScreen.cpp (hunk 2 above)
Comment (hrydgard) [reply UI/MemStickScreen.cpp:664]: Maybe should display a list of not-moved files? Not sure.

Comment (unknownbrackets) [reply UI/MemStickScreen.cpp:664]: Hm, maybe.  I guess as long as it says some larger files weren't moved... I wonder if some people might not even be able to access internal storage though?  Then again, how did the large files get there, then?  So probably okay...

-[Unknown]

File: Common/UI/View.cpp [hunk 4]
```diff
@@ -540,6 +540,11 @@ void InfoItem::Draw(UIContext &dc) {
 
 	UI::Style style = HasFocus() ? dc.theme->itemFocusedStyle : dc.theme->infoStyle;
 
+	if (choiceStyle_) {
+		style = dc.theme->buttonStyle;
```
Comment (unknownbrackets) [inline Common/UI/View.cpp:544]: nit: dc.theme->buttonFocusedStyle if focused?

-[Unknown]

Comment (hrydgard) [reply Common/UI/View.cpp:544]: Right

**Pull request merged**